kvm/requests.yaml

```diff
@@ -4,6 +4,6 @@ releases:
         - name: cert-exporter
```

Comment:
IMO you can get rid of all of these requests now that you're creating 16.2.0 which satisfies them. We would never go backwards in version following a minor.

Reply:
ah ok, i was going by the other providers - i thought they didn't delete the requests for old releases for a reason

Reply:
It depends on the requests. Like if it were `>= 16.1.0` then we might want to keep them in case we created `16.1.2`. But since these only apply to 16.2.0+ and you're creating that here, there's never a case where we'll create a release that doesn't satisfy the requests.